logging/bin/deploy_kibana_content.sh

```diff
@@ -76,16 +76,6 @@ kubectl -n logging wait pods --selector app=v4m-es,role=kibana --for condition=R
 set +e
 get_kb_api_url
 
-set -e
-
-if [ "$kibanaready" != "TRUE" ]; then
```

Comment:
Is this left over code from an earlier edit?  Why no corresponding changes in the PR?